packages/backend-common/src/reading/AwsS3UrlReader.test.ts

```diff
@@ -247,15 +247,18 @@ describe('AwsS3UrlReader', () => {
     });
 
     it('returns contents of an object in a bucket', async () => {
-      const response = await reader.read(
+      const { buffer } = await reader.readUrl(
         'https://test-bucket.s3.us-east-2.amazonaws.com/awsS3-mock-object.yaml',
       );
-      expect(response.toString().trim()).toBe('site_name: Test');
+      const response = buffer();
+      const fromStream = await response;
+
+      expect(fromStream.toString().trim()).toBe('site_name: Test');
```

Comment:
These could be simplified a bit more to get rid of some extra assignments :)
```suggestion
      const response = await buffer();
      expect(response.toString().trim()).toBe('site_name: Test');
```